scipy/stats/stats.py

```diff
@@ -346,6 +346,28 @@ def _broadcast_shapes_with_dropped_axis(a, b, axis):
     return shp
 
 
+def _convert_symmetric_p_value(pvalue, root_tail, alternative):
+    """
+    Convert a one-sided p-value from a symmetric distribution to a one or
+    two-sided p-value.
+    """
+    if alternative not in {"two-sided", "greater", "less"}:
+        raise ValueError("alternative should be "
+                         "'less', 'greater' or 'two-sided'")
+
+    if root_tail not in {"greater", "less"}:
+        raise ValueError("root_func should be 'less' or 'greater'")
+
+    if (alternative, root_tail) in {("greater", "greater"),
```

Comment:
This is if `alternative == root_tail`.

You can simplify this a lot if you start by checking for alternative two-sided, next check for the equality above, and end with `else`.

Reply:
See original comment about this [here](https://github.com/scipy/scipy/pull/12597#pullrequestreview-457179588)